src/components/app.jsx

```diff
@@ -25,24 +25,20 @@ const App = React.createClass({
           <div className="container" id="content" style={{marginBottom: 100}}>
             <section className="jumbotron" style={{marginTop: "1em"}}>
               <div className="row">
-                <div className="col-lg-3">
-                  <p>
-                    <img
-                      alt="Logo OpenFisca"
-                      className="img-responsive"
-                      src={url.resolve(config.websiteUrl, "/hotlinks/logo-openfisca.svg")}
-                    />
-                  </p>
+                <div className="col-md-3">
```

Comment:
Hmm, there's also `<p>{countryPackageName}@{countryPackageVersion}</p>` in this `<div>`, did you notice? 🤔